src/algebra/ordered_field.lean

```diff
@@ -655,4 +656,18 @@ by rw [abs_div, abs_one]
 lemma abs_inv (a : α) : abs a⁻¹ = (abs a)⁻¹ :=
 (abs_hom : monoid_with_zero_hom α α).map_inv' a
 
+-- TODO: add lemmas with `a⁻¹`.
+lemma one_div_strict_mono_decr_on : strict_mono_decr_on (λ x : α, 1 / x) (set.Ioi 0) :=
+λ x x1 y y1 xy, (one_div_lt_one_div (set.mem_Ioi.mp y1) (set.mem_Ioi.mp x1)).mpr xy
+
+lemma one_div_mono_exp (a1 : 1 ≤ a) {m n : ℕ} (mn : m ≤ n) :
+  1 / a ^ n ≤ 1 / a ^ m :=
+by refine (one_div_le_one_div _ _).mpr (pow_le_pow a1 mn);
+  exact pow_pos (lt_of_lt_of_le zero_lt_one a1) _
```

Comment:
Any reason not to express this with `monotone`? Something like
```suggestion
lemma one_div_mono_exp (a1 : 1 ≤ a) : monotone (λ n : ℕ, 1 / a ^ n) :=
λ m n mn, by refine (one_div_le_one_div _ _).mpr (pow_le_pow a1 mn);
  exact pow_pos (lt_of_lt_of_le zero_lt_one a1) _
```

Reply:
The very formal reason is that this function is not `monotone` it is "monotone non-increasing".  If you look very carefully, you will see that you assume `m ≤ n`, but the order of `m` and `n` is reversed in the inequality.  I guess that this could be monotone to `order_dual` but I am not sure that this would simplify matters.  Although, a lot of time has passed since I looked at the API for `order_dual`.

Should I try to do it with `order_dual`?  Or is there a `mono_decr` class of some sort?

Reply:
Darn it I make that mistake every time. I don't think `order_dual` is worth the trouble here.

Reply:
Ok, although I just pushed a commit with the change in that lemma and in the next one...

Should I revert it?